clients/src/main/java/org/apache/kafka/clients/consumer/ConsumerConfig.java

```diff
@@ -48,17 +48,22 @@
     public static final String MAX_POLL_RECORDS_CONFIG = "max.poll.records";
     private static final String MAX_POLL_RECORDS_DOC = "The maximum number of records returned in a single call to poll().";
 
+    /** <code>max.poll.interval.ms</code> */
+    public static final String MAX_POLL_INTERVAL_MS_CONFIG = "max.poll.interval.ms";
+    private static final String MAX_POLL_INTERVAL_MS_DOC = "The maximum delay between invocations of poll(). This is " +
```

Comment:
As an end-user the explanation of `liveness check` v.s. `consumer failures` is not distinguishing enough for these two cases. Could we just say `This is used to determine how long a consumer can be idle from fetching more records as it is used for processing last fetched records, or any other tasks between two consecutive poll calls...`?